doc/man3/EVP_EncryptInit.pod

```diff
@@ -1668,9 +1668,10 @@ This call is only valid when decrypting data.
 Where possible the B<EVP> interface to symmetric ciphers should be used in
 preference to the low-level interfaces. This is because the code then becomes
 transparent to the cipher used and much more flexible. Additionally, the
-B<EVP> interface will ensure the use of platform specific cryptographic
-acceleration such as AES-NI (the low-level interfaces do not provide the
-guarantee).
+B<EVP> interface will ensure the use of platform-specific cryptographic
+acceleration such as AES-NI and platform-provided data-independent timing
+protection such as DIT (the low-level interfaces do not provide these
```

Comment:
Not sure this should be making this generalized statement here.

Reply:
The point is that the low-level interfaces don't provide any such assurances. Would you be happier with something like

> B<EVP> interface will ensure the use of platform-specific cryptographic acceleration such as AES-NI and better protection against data-dependent instruction timings